src/Item/emote/types.ts

```diff
@@ -1,8 +1,21 @@
+import { StandardWearable } from '@dcl/schemas'
 import { WearableBodyShape } from '../wearable/types'
 
+export type Emote = Omit<StandardWearable, 'data'> & {
+  emoteDataADR74: {
```

Comment:
Here I copied the schema because we are currently using v4 of `@dcl/schemas` which doesn't have the new emote schema. I'd suggesting upgrading to the new version dhile doing https://github.com/decentraland/builder-server/issues/559 and then delete all the repeated types from this file and this [other one](https://github.com/decentraland/builder-server/blob/master/src/Item/emote/types.ts).